valhalla/baldr/edgeinfo.h

```diff
@@ -53,6 +53,15 @@ struct NameInfo {
   }
 };
 
+struct linguistic_text_header_t {
```

Comment:
This is our linguistic header.  It is used for pronunciations and in the future languages.